tests/phase0/block_processing/test_process_block_header.py

```diff
@@ -14,13 +16,93 @@
 pytestmark = pytest.mark.header
 
 
-def test_proposer_slashed(state):
+def test_sucess(state):
+    pre_state = deepcopy(state)
+    block = build_empty_block_for_next_slot(pre_state)
+
+    #
+    # setup pre_state to be ready for block transition
+    #
+    cache_state(pre_state)
+    advance_slot(pre_state)
+
+    post_state = deepcopy(pre_state)
+
+    #
+    # test block header
+    #
+    process_block_header(post_state, block)
+
+    return state, [block], post_state
```

Comment:
Pythonista speaking, it's a little weird to see that no assertion in this testcase. These three cases (`test_sucess`, `test_invalid_slot `, `test_invalid_previous_block_root`) can be combined into one testcase + maybe some setup. 😄

Reply:
ah right. it should for sure

Reply:
On second thought.. I'm not sure what we would assert. `process_block_header` actually has no side effects.